libs/output-mapping/tests/Writer/StorageApiLocalTableWriterTest.php

```diff
@@ -1298,6 +1296,72 @@ public function testAllowedDestinationConfigurations(
         self::assertSame($expectedTables, $tablesIds);
     }
 
+    #[NeedsEmptyOutputBucket]
+    public function testWriteAlwaysFlag(): void
+    {
+        if ($this->clientWrapper->getToken()->hasFeature(TableWriter::OUTPUT_MAPPING_SLICE_FEATURE)) {
+            self::markTestSkipped('Write always flag is not implemented for slice feature.');
+        }
+
+        $this->testAllowedDestinationConfigurations(
+            null,
+            null,
+            [
+                [
+                    'source' => 'table.csv',
+                    'destination' => '%s.table1',
+                    'write_always' => false,
+                ],
+                [
+                    'source' => 'table.csv',
+                    'destination' => '%s.table2',
+                    'write_always' => true,
+                ],
+            ],
+            ['%s.table2'],
+            true,
+        );
+    }
+
+    #[NeedsEmptyOutputBucket]
+    public function testAllowedMultipleMappingsOfSameSource(): void
+    {
+        if ($this->clientWrapper->getToken()->hasFeature(TableWriter::OUTPUT_MAPPING_SLICE_FEATURE)) {
```

Comment:
a nemůžem tyhle testy rovnou přesunout do StorageApiSlicedWriterTest a ten spouštět jen tam?

Reply:
No ale `StorageApiSlicedWriterTest` je starý test case a spouští se i na normálních projektech. Navíc tenhle test nic o slicování neříká.

Reply:
No jasně tak ne do toho, ale do jinýho, zkrátka, že by se to pouštělo jen v té slicer testusuitě a nebyly tu ty podmínky, ale to je asi jedno